src/screens/WeatherScreen.js

```diff
@@ -61,25 +75,30 @@ export const WeatherScreen = () => {
     <SafeAreaViewFlex>
       <ScrollView>
         {!!alerts?.length && (
-          <TitleContainer>
-            <Title>Wetterwarnungen</Title>
-          </TitleContainer>
+          <WrapperWithOrientation>
+            <TitleContainer accessibilityLabel={`${texts.weather.alertsHeadline} (Überschrift)`}>
+              <Title>{texts.weather.alertsHeadline}</Title>
+            </TitleContainer>
+            {device.platform === 'ios' && <TitleShadow />}
+            {alerts.map((alert, index) => (
+              <WeatherAlert
+                key={index}
+                description={alert.description}
+                event={alert.event}
+                start={alert.start}
+                end={alert.end}
+              />
+            ))}
+          </WrapperWithOrientation>
         )}
-        {!!alerts?.length &&
-          alerts.map((alert, index) => (
-            <WeatherAlert
-              key={index}
-              description={alert.description}
-              event={alert.event}
-              start={alert.start}
-              end={alert.end}
-            />
-          ))}
         {hasHourlyWeather(weatherMap) && (
           <View>
-            <TitleContainer>
-              <Title>Aktuelles Wetter</Title>
-            </TitleContainer>
+            <WrapperWithOrientation>
+              <TitleContainer accessibilityLabel={`${texts.weather.currentHeadline} (Überschrift)`}>
+                <Title>{texts.weather.currentHeadline}</Title>
+              </TitleContainer>
+              {device.platform === 'ios' && <TitleShadow />}
+            </WrapperWithOrientation>
             <FlatList
```

Comment:
I think it is awkward to have the hourly weather be wider than the rest.

I personally would just wrap the whole screen in the wrapper with orientation. What do you think?

Reply:
it is weird as well to have something scrolling horizontally kind of endless, but cut at some point on the screen without visual ending/border. we would have white space left and right and in the middle there are elements horizontally scrolling but "magically" wrapped. one would swipe the elements into and from the nowhere. that is why we keep the device frame as visual borders.

for that reason, we also have a special case for the carousel, where the prev and next images will be visible already with some shading, to be able to cover the whole width, as the element is "endless"

was this understandable enough?
can you think of

Reply:
i understand your point.

we can leave it as is then.

I want to point out though, that on your screenshot the cutoff still happens, just closer to the edge of the screen. I assume it has to do with some safe area inset.

Reply:
on devices with notch, yes. we cannot change that. but for devices without notch, it is the device edge.